standard/interfaces.md

```diff
@@ -305,75 +403,164 @@ These rules ensure that any covariant or contravariant usage of the interface re
+  > *Note*: As an interface cannot contain instance fields, an interface property cannot be an instance auto-property, as that would require the declaration of implicit hidden instance fields. *end note*
 
-### 19.4.4 Interface events
+- The type of an interface property shall be output-safe if there is a get accessor, and shall be input-safe if there is a set accessor.
+- An interface method declaration that has a block body or expression body as a *method_body* is `virtual`; the `virtual` modifier is not required, but is allowed.
+- An instance *property_declaration* that has no implementation is `abstract`; the `abstract` modifier is not required, but is allowed. It is *never* considered to be an automatically implemented property ([§15.7.4](classes.md#1574-automatically-implemented-properties)).
 
-Interface events are declared using *interface_event_declaration*s:
+### 19.4.4 Interface events
 
-```ANTLR
-interface_event_declaration
-    : attributes? 'new'? 'event' type identifier ';'
-    ;
-```
+This clause augments the description of events in classes [§15.8](classes.md#158-events) for events declared in interfaces.
 
-The *attributes*, *type*, and *identifier* of an interface event declaration have the same meaning as those of an event declaration in a class ([§15.8](classes.md#158-events)).
+Interface events are declared using *event_declaration*s ([§15.8.1](classes.md#1581-general)), with the following additional rules:
 
-The type of an interface event shall be input-safe.
+- *event_modifier* shall not include `override`.
+- A derived interface may implement an abstract interface event declared in a base interface (§15.8.5).
+- It is a compile-time error for *variable_declarators* in an instance *event_declaration* to contain any *variable_initializer*s.
+- An instance event with the `virtual` or `sealed` modifiers must declare accessors. It is *never* considered to be an automatically implemented field-like event ([§15.8.2](classes.md#1582-field-like-events)).
+- An instance event with the `abstract` modifier must not declare accessors.
+- The type of an interface event shall be input-safe.
 
 ### 19.4.5 Interface indexers
 
-Interface indexers are declared using *interface_indexer_declaration*s:
+This clause augments the description of indexers in classes [§15.9](classes.md#159-indexers) for indexers declared in interfaces.
 
-```ANTLR
-interface_indexer_declaration
-    : attributes? 'new'? type 'this' '[' parameter_list ']'
-      '{' interface_accessors '}'
-    | attributes? 'new'? ref_kind type 'this' '[' parameter_list ']'
-      '{' ref_interface_accessor '}'
-    ;
-```
+Interface indexers are declared using *indexer_declaration*s ([§15.9](classes.md#159-indexers)), with the following additional rules:
+
+- *indexer_modifier* shall not include `override`.
+- An *indexer_declaration* that has an *expression body* or contains an accessor with a body is `virtual`; the `virtual` modifier is not required, but is allowed.
```

Comment:
Maybe:

```suggestion
- An *indexer_declaration* that has an expression body or contains an accessor with a block body is `virtual`; the `virtual` modifier is not required, but is allowed.
```

in other clauses here expression body is not emphasised and block body is used.

Reply:
This is the one comment where we crossed. They are all consistent now. You can resolve this is you're satisfied with the change.

Reply:
@BillWagner I’m not seeing changes on this line or similar ones on the other clauses so they remain inconsistent. I've reloaded the page etc. so has GitHub messed up?